src/check/package-json.test.js

```diff
@@ -28,7 +40,19 @@ tap.ok(
   'package.json has "author" property'
 );
 
-tap.equal(pkg.main, 'es5/plugin.js', 'package.json "main" is "es5/plugin.js"');
+tap.equal(pkg.main, 'dist/es5/index.js', 'package.json "main" is "dist/es5/index.js"');
+tap.equal(pkg['jsnext:main'], 'src/js/index.js', 'package.json "main" is "src/js/index.js"');
+
+tap.ok(
+  isNonEmptyString(pkg['generator-videojs-plugin'].version),
+  'package.json "generator-videojs-plugin.version" exists'
+);
+
+tap.ok(_.isPlainObject(pkg.spellbook), 'package.json has "spellbook" object');
+
+Object.keys(pkg.spellbook).forEach(k => {
+  tap.equal(typeof pkg.spellbook[k], 'boolean', `package.json "spellbook.${k}" is a boolean`);
```

Comment:
* `src`, `js`, `css`, `test`, and `docs` can be `false` or an `object`. If they are an empty object its the same as true but they can have [individual `src` folder settings](https://github.com/videojs/spellbook/blob/master/src/utils/get-config.js#L76-L91). This is outside the scope of the generator, but its possible.
*  I also noticed, looking back at the code, that there is [one Array setting too](https://github.com/videojs/spellbook/blob/master/src/utils/get-config.js#L66).